data/icij-example/suspicious.txt

```diff
@@ -0,0 +1,17 @@
+Andrej Babis
+Sotheby’s International Realty
+Chateau Bigaud
+Agrofert
+Arzu Aliyeva
+Ilham Aliyev
+Mossack Fonseca
+Fazil Mammadov
+AtaHolding
+FM Management Holding Group S.A. Stand
+UF Universe Foundation
+Mehriban Aliyeva
+Heydar Aliyev
+Leyla Aliyeva
+AtaHolding Azerbaijan
+Financial Management Holding Limited
+Hughson Management Inc.
```

Comment:
I manually "label" suspicious entities for my 2 articles, and use that to filter the Senzing KB